cypress/integration/recieve-notice/recieve-notice-after-checkout.spec.js

```diff
@@ -0,0 +1,133 @@
+import testTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import checkInActions from '../../support/fragments/check-in-actions/checkInActions';
+
+// TODO email checking
+describe('Recieving notice: Checkout', () => {
+  const ITEM_BARCODE = generateItemBarcode();
+  const patronGroup = {};
+  const userData = {
+    username: `Test_user_${getRandomPostfix()}`,
+    active: true,
+    barcode: uuid(),
+    personal: {
+      preferredContactTypeId: '002',
+      lastName: `Test user ${getRandomPostfix()}`,
+      email: 'test@folio.org',
+    },
+    departments: []
+  };
+  let templateId;
+  let noticePolicyId;
+  let loanType;
+  let materialType;
+  let location;
+  let holdingType;
+  let holdingSource;
+  let instanceType;
+  let userServicePoint;
+
+  beforeEach('Creating help entities', () => {
+    cy.getAdminToken()
+      .then(() => {
+        patronGroups.createViaApi();
+      })
+      .then(res => {
+        cy.createUserApi({
+          patronGroup: res.id,
+          ...userData
+        }).then((createdUser) => {
+          userData.id = createdUser.id;
+        });
+        patronGroup.name = res.group;
+        patronGroup.id = res.id;
+      })
+      .then(() => {
+        cy.getServicePointsApi({ limit: 1, query: 'pickupLocation=="true"' }).then((res) => {
+          cy.addServicePointToUser(res[0].id, userData.id).then((points) => {
+            userServicePoint = points.body.defaultServicePointId;
+            console.log(points);
+          });
+        });
+      })
+      .then(() => {
+        cy.getMaterialTypes({ limit: 1 }).then((res) => { materialType = res.id; });
+        cy.getLocations({ limit: 1 }).then((res) => { location = res.id; });
+        cy.getHoldingTypes({ limit: 1 }).then((res) => { holdingType = res[0].id; });
+        cy.getHoldingSources({ limit: 1 }).then((res) => { holdingSource = res[0].id; });
+        cy.getInstanceTypes({ limit: 1 }).then((res) => { instanceType = res[0].id; });
+        cy.getLoanTypes({ limit: 1 }).then((res) => { loanType = res[0].id; });
+      })
+      .then(() => {
+        cy.createInstance({
+          instance: {
+            instanceTypeId: instanceType,
+            title: `Pre-checkout instance ${Number(new Date())}`,
+          },
+          holdings: [{
+            holdingsTypeId: holdingType,
+            permanentLocationId: location,
+            sourceId: holdingSource,
+          }],
+          items: [
+            [{
+              barcode: ITEM_BARCODE,
+              missingPieces: '3',
+              numberOfMissingPieces: '3',
+              status: { name: 'Available' },
+              permanentLoanType: { id: loanType },
+              materialType: { id: materialType },
+            }],
+          ],
+        });
+      })
+      .then(() => {
+        noticePolicyTemplate.createViaApi(TEMPLATE_CATEGORIES.loan).then(res => { templateId = res.body.id; });
```

Comment:
"Click on the template name in the list of patron notice templates" - looks like created patron notice template should be verified in GUI